### PR TITLE
Article: React Router Cheatsheet

### PR DESCRIPTION
**Pull request by awesomeaniruddh**

### CODE REVIEW

File: React-Router-Cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# React Router Cheatsheet
+
+Based on [http://ricostacruz.com/cheatsheets/react-router.html](http://ricostacruz.com/cheatsheets/react-router.html) but updated for v2.0.0 of React Router.
+
+### Basic
```
Comment (abhisekp) [inline React-Router-Cheatsheet.md]: All the major relevant heading apart from the first relevant/major heading must be Heading 2 as @camperbot delimits at each Heading 2 tag i.e. `##`

**Comment by abhisekp** [comment]
# :+1: LGTM *for formatting*

content needs to be reviewed by @alayek or someone who knows React.

File: React-Router-Cheatsheet.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+Based on [http://ricostacruz.com/cheatsheets/react-router.html](http://ricostacruz.com/cheatsheets/react-router.html) but updated for v2.0.0 of React Router.
+
+## Basic
+
+```js
+import { render } from 'react-dom'
+import { Router, Route, browserHistory } from 'react-router'
+
+// Import RootView and NoMatch
```
Comment (alayek) [inline React-Router-Cheatsheet.md]: Any reason you are putting these in comments and not actually importing?

Comment (awesomeaniruddh) [inline React-Router-Cheatsheet.md]: Because they're components created by the developer

**Comment by Rafase282** [comment]
When @alayek gives the okay.

**Comment by alayek** [comment]
# LGTM :+1:

**Pull request merged**